pkg/config/engine/api.go

```diff
@@ -24,6 +24,7 @@ type Interface interface {
 	RemoveRuntime(string) error
 	Save(string) (int64, error)
 	GetRuntimeConfig(string) (RuntimeConfig, error)
+	EnableCDI()
```

Comment:
I think `Set` is only used to enable CDI. Let's remove that from the interface if that's the case.